src/System.Buffers.ReaderWriter/System.Buffers.ReaderWriter.csproj

```diff
@@ -2,16 +2,16 @@
   <Import Project="..\..\tools\common.props" />
   <PropertyGroup>
     <Description>Buffer Readers, Writers, and Transformations</Description>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFramework>netstandard1.1</TargetFramework>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <PackageTags>Span BufferReader BufferWriter corefxlab</PackageTags>
   </PropertyGroup>
   <ItemGroup>
-    <ProjectReference Include="..\System.Collections.Sequences\System.Collections.Sequences.csproj" />
-    <ProjectReference Include="..\System.Text.Primitives\System.Text.Primitives.csproj" />
     <ProjectReference Include="..\System.Text.Utf8String\System.Text.Utf8String.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="System.IO.Pipelines" Version="$(SystemIOPipelinesVersion)" />
```

Comment:
ReaderWriter shouldn't depend on the higher level library (it should be the other way around).